packages/forms/src/model.ts

```diff
@@ -2091,19 +2091,19 @@ export class FormArray extends AbstractControl {
    * ]);
    * arr.reset(['name', 'last name']);
    *
-   * console.log(this.arr.value);  // ['name', 'last name']
+   * console.log(arr.value);  // ['name', 'last name']
    * ```
    *
    * ### Reset the values in a form array and the disabled status for the first control
    *
    * ```
-   * this.arr.reset([
+   * arr.reset([
    *   {value: 'name', disabled: true},
    *   'last'
    * ]);
    *
-   * console.log(this.arr.value);  // ['name', 'last name']
-   * console.log(this.arr.get(0).status);  // 'DISABLED'
+   * console.log(arr.value);  // ['last']
+   * console.log(arr.get('0').status);  // 'DISABLED'
```

Comment:
I think we can use `at` here:
```suggestion
   * console.log(arr.at(0).status);  // 'DISABLED'
```

Reply:
FYI since it was a minor update I've applied this change and marked this PR as ready for merge.